### PR TITLE
`hardhat-zksync-vyper` docs

### PR DESCRIPTION
**Pull request by ly0va**

### CODE REVIEW

File: docs/api/hardhat/getting-started.md [hunk 1]
```diff
@@ -2,14 +2,16 @@
...
-- `@matterlabs/hardhat-zksync-solc` for smart contract compilation.
+- `@matterlabs/hardhat-zksync-solc` for smart contract compilation using Solidity.
+- `@matterlabs/hardhat-zksync-vyper` for smart contract compilation using Vyper.
 - `@matterlabs/hardhat-zksync-deploy` for smart contract deployment.
 
 To learn more about hardhat itself, check out their [documentation](https://hardhat.org/getting-started/).
 
-This tutorial shows how to set up a zkSync hardhat project from scratch.
+This tutorial shows how to set up a zkSync solidity project using hardhat from scratch.
+If you are using vyper, check out the [reference](./reference.md#matterlabs-hardhat-zksync-vyper) for our vyper plugin or the [example](https://github.com/matter-labs/hardhat-zksync/tree/main/examples/vyper-example) in our github repo!
```
Comment (hedgar2017) [inline docs/api/hardhat/getting-started.md:14]: Please capitalize the names of the languages everywhere.

**Review by hedgar2017 — CHANGES_REQUESTED** [review]

File: docs/api/hardhat/getting-started.md [hunk 2]
```diff
@@ -2,14 +2,16 @@
...
-- `@matterlabs/hardhat-zksync-solc` for smart contract compilation.
+- `@matterlabs/hardhat-zksync-solc` for smart contract compilation using Solidity.
+- `@matterlabs/hardhat-zksync-vyper` for smart contract compilation using Vyper.
 - `@matterlabs/hardhat-zksync-deploy` for smart contract deployment.
 
 To learn more about hardhat itself, check out their [documentation](https://hardhat.org/getting-started/).
 
-This tutorial shows how to set up a zkSync hardhat project from scratch.
+This tutorial shows how to set up a zkSync Solidity project using hardhat from scratch.
+If you are using Vyper, check out the [reference](./reference.md#matterlabs-hardhat-zksync-vyper) for our vyper plugin or the [example](https://github.com/matter-labs/hardhat-zksync/tree/main/examples/vyper-example) in our github repo!
```
Comment (barakshani) [inline docs/api/hardhat/getting-started.md:14]: vyper or Vyper?

File: docs/api/hardhat/reference.md [hunk 3]
```diff
@@ -45,21 +43,73 @@ networks: {
...
 
-If a certain contract is a dependency for other smart contracts, it may appear in multiple flattened files; thus, it will appear multiple times in the `artifacts` folder. During deployment, it may be required to specify the full path to the contract artifact to the `Deployer` class instance. The deployer script will provide hints on what paths may be used. If this contract has a unique name in the project, it is possible choose any of the proposed ones.
+This plugin is used to provide a convenient interface for compiling zkSync smart contracts using Vyper.
+
+### Npm
+
+[@matterlabs/hardhat-zksync-vyper](https://www.npmjs.com/package/@matterlabs/hardhat-zksync-vyper)
+
+This plugin is used in conjunction with [@nomiclabs/hardhat-vyper](https://www.npmjs.com/package/@nomiclabs/hardhat-vyper).
+To use it, you have to install and import both of them.
```
Comment (barakshani) [inline docs/api/hardhat/reference.md:66]: "you have to install and import both." ?

File: docs/api/hardhat/reference.md [hunk 4]
```diff
@@ -45,21 +43,73 @@ networks: {
...
+
+Add the latest version of this plugin to your project with the following command:
```
Comment (barakshani) [inline docs/api/hardhat/reference.md:68]: Something in this sentence is missing

**Review by perekopskiy — APPROVED** [review]

**Pull request merged**